Sources/RegexBuilder/DSL.swift

```diff
@@ -10,6 +10,16 @@
 //===----------------------------------------------------------------------===//
 
 import _MatchingEngine
+@_spi(RegexBuilder) import _StringProcessing
+import _StringProcessing
```

Comment:
Do we need this given we already have line 13?

Reply:
Good catch. Removed.